src/index.html

```diff
@@ -8,7 +8,70 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="logo__container">
+        <a href="#" class="logo__link">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="image"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >APPLE</a>
```

Comment:
Fix indents for all links like this:
```suggestion
            <a
              href="#"
              class="nav__link is-active"
            >
              APPLE
            </a>
```